src/qibolab/instruments/zhinst.py

```diff
@@ -673,7 +677,8 @@ def play_sweep_select_dual(exp, qubit, pulse, section, parameters):
             for sweeper in pulse.zhsweepers:
                 if sweeper.uid == "amplitude":
                     sweeper_amp_index = pulse.zhsweepers.index(sweeper)
-                    sweeper.values = sweeper.values.copy()
+                    # sweeper.values = sweeper.values.copy()
+                    # FIXME: Dont change the values coming back to qibocal
```

Comment:
Why did you remove the sweepers.copy if you still have this problem? 
It could lead to really unexpected behaviour and be difficult to debug

Reply:
@maxhant wanted it because it was doing loops on the qibocal routine. Now that is not necessary so it should not be an issue but I still want to play with it when the fridge is ok.

Reply:
I would still use the deep copy... I also have it in the rfsoc driver, to be sure that any change inside the driver do not propagate outside.
Anyway, if you don't think it's useful, it's ok for me to approve

Reply:
We can keep it, maybe in the future someone may want to still do software sweepers for some reason.